frontend/src/components/transactions/LineItemTable.tsx

```diff
@@ -63,18 +81,23 @@ export default function LineItemTable() {
             {table.getRowModel().rows?.length ? (
               table.getRowModel().rows.map((row) => (
                 <TableRow
-                  key={row.id}
-                  data-state={row.getIsSelected() && "selected"}
+                key={row.id}
+                data-state={row.getIsSelected() && "selected"}
                 >
-                  {row.getVisibleCells().map((cell) => (
-                    <TableCell key={cell.id}>
+                {row.getVisibleCells().map((cell) => (
+                  <TableCell key={cell.id}>
                       {flexRender(
                         cell.column.columnDef.cell,
                         cell.getContext()
                       )}
-                    </TableCell>
-                  ))}
-                </TableRow>
+                  </TableCell>
+                ))}
+                <TableCell>
+                  <Button onClick={() => openReconcileDialog(row)}>
```

Comment:
```suggestion
                  <Button onClick={() => openReconcileDialog(row)} variant="ghost">
```

this will make the button not green and more chill

Reply:
This has been replaced by the little arrow boy now!